lib/realtime/tenants.ex

```diff
@@ -78,8 +69,10 @@ defmodule Realtime.Tenants do
       nil ->
         {:error, :tenant_not_found}
 
-      {:ok, _health_conn} ->
+      {:ok, health_conn} ->
         connected_cluster = UsersCounter.tenant_users(external_id)
+        %{extensions: [%{settings: settings} | _]} = Cache.get_tenant_by_external_id(external_id)
+        Helpers.transaction(health_conn, fn _ -> Migrations.run_migrations(settings) end)
```

Comment:
Needs to run even if db_connected is false

Reply:
made the change, not sure if the best approach